format_cef/cef.py

```diff
@@ -125,15 +125,59 @@ def sanitise(t, debug_name):
 
 Extension = namedtuple('Extension', ('key_name', 'sanitiser'))
 ipv4_addr = str_sanitiser(r'\.'.join(['\d{1,3}'] * 4))
+ipv6_addr = str_sanitiser(r'\:'.join(['[0-9a-fA-F]{1,4}'] * 8))  # only complete ipv6 address accepted
+mac_addr = str_sanitiser(r'\:'.join(['[0-9a-fA-F]{2}'] * 6))
 str_31 = str_sanitiser(max_len=31)
 str_63 = str_sanitiser(max_len=63)
+str_128 = str_sanitiser(max_len=128)
+str_255 = str_sanitiser(max_len=255)
 str_1023 = str_sanitiser(max_len=1023)
 
-# An incomplete list of valid CEF extensions
 valid_extensions = {
     'applicationProtocol': Extension('app', str_31),
+    'baseEventCount': Extension('cnt', int_sanitiser()),
```

Comment:
@gergelyv78 did you manually write out this table by eye, or is there a machine readable copy of the spec somewhere?

Reply:
I did it by eye. Although I tried to double check the list, but there is a chance that a few misspell were made.
Even the guide looks failing in standardization, as the CEF Key Names mostly begins with small capitals except "C6a4Label" or "DevicePayloadId". I'm not sure if I followed standard all the time.

Reply:
Updated all the exceptions: "C6a4Label", "DevicePayloadId" and "DeviceOutboundInterface"